src/js/components/Form/stories/ValidateOnChange.js

```diff
@@ -12,36 +11,38 @@ export const ValidateOnChange = () => (
           validate="change"
           onReset={event => console.log(event)}
           onSubmit={({ value }) => console.log('Submit', value)}
+          onValidate={validationResults => {
```

Comment:
I think this example has improved with these changes, but it does not demonstrate the whole `valid` story. For users to be able to leverage this new functionality, a real use case example of how this prop could be used will be helpful. For example, you can change the story so `valid` will have an effect on the chain of events, from what I understand a common use case for your team is to disable the Submit button when valid is false. It will be nice to see how this end-to-end flow is working on the story.

Reply:
> Can you please add more details why you think this PR has breaking changes, quoting from the description above:
> 
> > Is this change backwards compatible or is it a breaking change?
> > Yes

Meant to say this change should be compatible, not a breaking change.

Reply:
Will add. Makes sense.